webapp/src/blocks/block.ts

```diff
@@ -5,7 +5,7 @@ import difference from 'lodash/difference'
 
 import {Utils} from '../utils'
 
-const contentBlockTypes = ['text', 'image', 'divider', 'checkbox'] as const
+const contentBlockTypes = ['text', 'image', 'divider', 'checkbox', 'attachment'] as const
```

Comment:
Attachments are not "contentBlocks". They are their own type.

Reply:
Got it.